src/pages/about/governance.md

```diff
@@ -1,44 +1,51 @@
----
-id: governance
----
-
 # Tauri Governance
 
-## Sustainability
+One of the main goals of the organizational structure of Tauri is to guarantee the sustainability, health and well-being of its contributors. [The Tauri Programme within the Commons Conservancy](https://dracc.commonsconservancy.org/0035/) was established to facilitate an open, transparent and efficient governance process throughout the future development of the Tauri and its auxiliary materials.
 
-One of the main goals of the organizational structure of Tauri is to guarantee the sustainability of Tauri and the health and well-being of its contributors. The world of Open Source is fraught with peril and discord, and we have taken measures to ensure the longevity of Tauri. This document explains how we go about doing so.
+## Tauri Working Group
 
-## Organizational Structure
+The Tauri Working Group is the collective framework created to enable this governance process. It's composed of the following components:
```

Comment:
Did we ever find a great description for the working group? I think the most *truthful* is that it's where regular contributors more deeply involved with Tauri collaborate.

Reply:
Eh, this is one that I think is slightly evolving over time and will probably continue to evolve before it settles a bit more. So I'm not sure if we want to spend too much time on it in this current moment.

Reply:
Yep I got stuck on this for like an hour the other day. Somewhat leaning towards, *it's where we coordinate for long term goals.* But indeed. I'd rather have a meh definition right now, and concentrate on getting straight inaccurate things corrected.

(Code Teams are superseded, and there's no WG Media, sort of thing :joy:)